src/components/views/elements/Highlight.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2017 Michael Telatynski <7t3chguy@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {highlightBlock} from 'highlight.js';
+
+export default class Highlight extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        children: PropTypes.node,
```

Comment:
We don't do this anywhere, but having just seen [this thread](https://github.com/yannickcr/eslint-plugin-react/issues/7), I'd be happy to start doing this for clarity of "this component uses its children, see it's in the prop types". (and IRLd with @dbkr and he agreed, although we won't obsess over it.)